### PR TITLE
Implement find tag command

### LINKED ISSUE #4: Search tag
As a Student, I can search for students by their tags so I can categorise them.

### PR DESCRIPTION
**Pull request by yuechen2001**
Fixes #4 

- [x] Add a new feature to let users search for contacts by their tags
- [x] Update UI and wireframes on Figma
- [x] Update DG and UG

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FindTagCommand.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import seedu.address.model.Model;
+import seedu.address.model.person.TagsContainKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose tag contains any of the argument keywords.
+ * Keyword matching is case-insensitive.
+ */
+public class FindTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "findTag";
```
Comment (angsongyee) [inline src/main/java/seedu/address/logic/commands/FindTagCommand.java:18]: Using camel case for commands looks quite odd. Maybe `findtag` or `find-tag` would work better instead.

File: src/test/java/seedu/address/model/person/TagsContainKeywordsPredicateTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+        // different person -> returns false
+        assertFalse(firstPredicate.equals(secondPredicate));
+    }
+
+    @Test
+    public void test_nameContainsKeywords_returnsTrue() {
+        // One keyword
+        TagsContainKeywordsPredicate predicate = new TagsContainKeywordsPredicate(Collections.singletonList("Alice"));
+        assertTrue(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
```
Comment (angsongyee) [inline src/test/java/seedu/address/model/person/TagsContainKeywordsPredicateTest.java:46]: The tests here don't match up with the command's expected behavior.

**Review by angsongyee — CHANGES_REQUESTED** [review]

**Review by angsongyee — APPROVED** [review]
LGTM

**Review by maiyasaliha — APPROVED** [review]
LGTM

**Pull request merged**